MacrobenchmarkSample/README.md

```diff
@@ -0,0 +1,28 @@
+## Macrobenchmark Sample
+
+This sample project shows how to use the Jetpack MacroBenchmark library.
+
+### Running
+
+Open the project in Android Studio 3.4 or later, and run benchmarks as you
+usually would run tests: Ctrl-Shift-F10 (Mac: Ctrl-Shift-R)
+
```

Comment:
Link to all the files, like in BenchmarkSample/README.md

Also link to the docs at the end:

```
See the [Macrobenchmark guide](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/benchmark/docs/macrobenchmark.md) for more information on the library

```

Reply:
See the first part, I'd like to see something like the BenchmarkSample's list of test files to tell people where to start looking:

```
* [TrivialStartupBenchmark](...) measures the startup time of a trivial Activity, with a single TextView
* [SmallListStartupBenchmark](...) measures the startup time with a simple RecyclerView
* [FrameTimingBenchmark](...) scrolls a sample RecyclerView, measuring frame timing / jank.
```